docker/Dockerfile.cpu

```diff
@@ -3,23 +3,25 @@ FROM ubuntu:18.04
 ENV PYTHONUNBUFFERED TRUE
 
 RUN apt-get update && \
-    DEBIAN_FRONTEND=noninteractive apt-get install -y \
-    build-essential \
-    libatlas-base-dev \
-    libopencv-dev \
-    graphviz \
-    python-dev \
+    DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y \
+    fakeroot \
+    ca-certificates \
+    dpkg-dev \
+    g++ \
     python3-dev \
     openjdk-8-jdk-headless \
     curl \
     vim \
     && rm -rf /var/lib/apt/lists/* \
     && cd /tmp \
     && curl -O https://bootstrap.pypa.io/get-pip.py \
-    && python get-pip.py
+    && python3 get-pip.py
+
+RUN update-alternatives --install /usr/bin/python python /usr/bin/python3 1
```

Comment:
Why do we need update-alternatives?

Reply:
python3-dev only install python3 command line, there is no /usr/bin/python exists

Reply:
I think I should understand this tool well. So, is this just creating a sym_link?